src/python/pants/bin/pyoxidizer-howto.md

```diff
@@ -1,6 +1,6 @@
 NOTES:
 
-Run `./pants --no-pantsd --no-remote-cache-write package --pyoxidizer-args="--target-triple=aarch64-apple-darwin" --pyoxidizer-interpreter-constraints="['CPython==3.9.*']" src/python/pants/bin:pants_oxidized_experimental`
+Run `./pants package --pyoxidizer-interpreter-constraints="['CPython==3.9.*']" src/python/pants/bin:pants_oxidized_experimental`
```

Comment:
are the ICs only because you're on Apple Silicon, or it's to force the target to use 3.9? Should we set this in pants.toml?

Reply:
At the moment, the pants scripts will compile `native_engine` against the Python SDK that is specified by Pants' global interpreter constrains. PyOxidizer will eventually package the Pants Python code along with a Python distribution. 

At the moment, our interpreter constraints don't naturally align. If this were for anything other than packaging our own product, it might be worth fixing properly, but I suspect this will end up just being part of a release script.